### DISCUSSION TITLE
GUI

### DISCUSSION THREAD

**Discussion by sajjadarashhh** [CosmosOS/Cosmos#2255] [Q&A]
There's any way to we make GUI for cosmos os with .net 5 or .net 6?

**Comment by sajjadarashhh** [comment]
Do you have an idea!? 
@quajak

**Reply by quajak** [reply]
There is a GUI demo kernel and a test kernel for ui methods which can help you get started. You can also take a look at https://cosmosos.github.io/articles/Kernel/CGS.html
